ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.7-aws-ovn.yaml

```diff
@@ -0,0 +1,201 @@
+  openshift_knative-v0.22.0_knative-serving-src:
+    name: knative-v0.22.0
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.22.3_knative-eventing-kafka-src:
+    name: knative-v0.22.3
+    namespace: openshift
+    tag: knative-eventing-kafka-src
+  openshift_release_golang-1.14:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+  openshift_release_golang-1.15:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_knative-v0.22.0_knative-eventing-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.22.0:knative-eventing-src
+    openshift_knative-v0.22.0_knative-serving-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.22.0:knative-serving-src
+    openshift_knative-v0.22.3_knative-eventing-kafka-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.22.3:knative-eventing-kafka-src
+  to: serverless-operator-src
+- context_dir: .
+  dockerfile_path: serving/ingress/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.15:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.15
+  to: knative-openshift-ingress
+- context_dir: .
+  dockerfile_path: knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.15:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.15
+  to: knative-operator
+- context_dir: .
+  dockerfile_path: openshift-knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.15:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.15
+  to: openshift-knative-operator
+- context_dir: olm-catalog/serverless-operator
+  from: base
+  to: serverless-bundle
+- context_dir: olm-catalog/serverless-operator
+  dockerfile_path: stopbundle.Dockerfile
+  from: base
+  to: serverless-stop-bundle
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.7"
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws-ovn-ocp-47-continuous
+  cron: 0 0 * * 0
+  steps:
+    cluster_profile: aws
+    test:
+    - as: serverless-e2e
+      commands: make install-mesh test-e2e-with-kafka
+      dependencies:
+      - env: KNATIVE_OPENSHIFT_INGRESS
+        name: knative-openshift-ingress
+      - env: KNATIVE_OPERATOR
+        name: knative-operator
+      - env: OPENSHIFT_KNATIVE_OPERATOR
+        name: openshift-knative-operator
+      from: serverless-operator-src
+      resources:
+        limits:
+          memory: 6Gi
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: openshift-e2e-aws-ovn
```

Comment:
This line brings OVN to the usual AWS installation. The rest of the files are not specific for OVN